Idfix

khaefeli opened the pull request:
#35 allow setting the user id over the resource. 
and workaround the issue with empty usernames &  the default anonymous user (be careful here!)

Commit:
allow setting the user id over the resource

Commit:
workaround empty username / anonymous user

logicminds commented:
Thanks for the flurry of PRs.  I'll need to update the travis file unless you can fix the .travis in a new PR.  Also since these changes require Facter with structured facts we need to communicate that in the readme and metadata file.  We should be targeting Puppet 4.x+ anyways.  Please create a CHANGELOG.md and CONTRIBUTORS.md.  Add your fixes to Changelog and throw your name/email in the CONTRIBUTORS file.

Commit:
remove ruby 1.9.3 support because it's EOL

khaefeli commented:
I've updated the `.travis.yml` and removed the ruby 1.9.3 support.
tests are running now, but many errors ;)

I can create the changelog and contributors files, should we target v 1.0 and support Puppet 4.x+  without or without older versions?

logicminds commented:
Please target a 1.0 release

logicminds commented:
We can also target ruby 2.1.9 and ruby 2.4.3
